frontends/mit-learn/src/page-components/Dialogs/AddToListDialog.tsx

```diff
@@ -130,59 +122,61 @@ const AddToListDialogInner: React.FC<AddToListDialogInnerProps> = ({
   })
 
   return (
-    <Dialog
+    <FormDialog
```

Comment:
I had considered using `FormDialog`, but at first it didn't seem necessary since this is barely a form and there's only one input element. This makes sense though, it cleans things up and `onReset` fixes our issue with the selections persisting.